app/classifier/tasks/text/index.cjsx

```diff
@@ -48,8 +48,10 @@ module.exports = React.createClass
       annotation.value isnt '' or not task.required
 
   getDefaultProps: ->
-    task: null
-    annotation: null
+    task:
+      instruction: ''
+    annotation:
+      value: null
```

Comment:
Yeah, I made annotations a required property in the task navigation, but it turns out they aren't always defined when a subject first loads. Nothing in `TaskNav` will crash if that value isn't supplied. Validating annotations is a bit of a nightmare.